nvflare/apis/impl/controller.py

```diff
@@ -979,10 +988,12 @@ def _check_dead_clients(self):
     def _client_still_alive(self, client_name):
         now = time.time()
         report_time = self._dead_client_reports.get(client_name, None)
+        grace_period = ConfigService.get_float_var(name=_CONFIG_VAR_DEAD_CLIENT_GRACE_PERIOD, default=30.0)
```

Comment:
just a note, in the future it might be good if we have a common place to keep these default values for timing so that people can just check that

Reply:
Yes, I'm thinking the same thing. We'll need to centrally manage all such control vars so they can be easily documented and explained to other people.